samples/excel/45-named-item/create-and-use-named-item-for-range.yaml

```diff
@@ -0,0 +1,171 @@
+id: excel-create-and-use-named-item-for-range
+name: Create and use named range item
+description: Create and use named range item
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |-
+        $("#add-name").click(() => tryCatch(addName));
+        $("#format-named-range").click(() => tryCatch(formatRangeByName));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function addName() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const headerRange = sheet.getRange("A1:E1");
+                sheet.names.add("ExpensesHeader", headerRange);
+                const namedItems = sheet.names.load("name, type");
+
+                await context.sync();
+
+                let namedItemsList = `This workbook contains ${namedItems.items.length} named item(s):`;
+                for (let i = 0; i < namedItems.items.length; i++) {
```

Comment:
In which case the whole thing can become a single statement.

    utility.notify(<title>,
        `This workbook ... named item(s): ` +
        namedItems.items.map(...).join(", ")